src/test/HostActivationTests/GivenThatICareAboutDotnetArgValidationScenarios.cs

```diff
@@ -94,7 +94,7 @@ public void Detect_Missing_Argument_Value()
             dotnet.Exec("--fx-version")
                 .CaptureStdOut()
                 .CaptureStdErr()
-                .Execute()
```

Comment:
This change is unrelated to the environment variable change; however still address test usability via #4099